SiN MZI designs

DilshaAppuHennadi opened the pull request:
Design submissions for SiN run with ANT.

DilshaAppuHennadi committed:
SiN MZI designs

lukasc-ubc commented:
I notice that you have a box on the Oxide open layer.  Please note that this fabrication run does not include this.

I also notice that your MZIs containing the SiN_perf_WG_R75 are perfectly balanced. This means that you will not see an FSR in the response.

DilshaAppuHennadi commented:
Hi, I can have an updated layout uploaded today.

DilshaAppuHennadi committed:
Merge branch 'SiEPIC:main' into main

DilshaAppuHennadi committed:
Updated SiN layouts

Files A-I contain 2 MZI structures - 1 balanced (both arms perf'd), 1 unbalanced (1 arm perf'd, 1 arm normal). File J contains a balanced MZI with no perfs as reference and a loop back structure.
